tests/test_image_seq2seq.py

```diff
@@ -103,7 +105,41 @@ def test_multitask(self):
 
         valid, test = testing_utils.train_model(args)
         self.assertLessEqual(
-            valid['ppl'], 5.0, 'failed to train image_seq2seq on image+text task',
+            valid['ppl'], 5.0, 'failed to train image_seq2seq on image+text task'
+        )
+
+    @testing_utils.retry(ntries=3)
```

Comment:
Great to have a check here! Whenever I write a PR I need to have a little popup that tells me, "Write a dang check"...